src/core/MREntity.js

```diff
@@ -307,13 +307,13 @@ export class MREntity extends MRElement {
      * @function
      * @description Callback function of MREntity - does nothing. Is called by the connectedCallback.
      */
-    connected() {}
+    async connected() {}
 
     /**
      * @function
      * @description The connectedCallback function that runs whenever this entity component becomes connected to something else.
```

Comment:
can you add `(async)` to the description like you did for the other newly async functions?